coredns/tests/conftest.py

```diff
@@ -18,7 +18,8 @@
 PORT = '9153'
 URL = "http://{}:{}/metrics".format(HOST, PORT)
 
-DIG_ARGS = ["dig", "google.com", "@127.0.0.1", "-p", "54"]
+# One lookup each for the forward and proxy plugins
+DIG_ARGS = ["dig", "google.com", "@127.0.0.1", "example.com", "@127.0.0.1", "-p", "54"]
```

Comment:
Can you pass "HOST" instead of "127.0.0.1" ?

Reply:
Done